tests/xml.js

```diff
@@ -150,6 +152,18 @@ define([
             eq('<output attr=value />', '<output attr="value" />', false);
         });
 
+        it("should fail on escape unquoted attribute value with <", function () {
+            chai.expect(function () {
+                eq('<output value=2<3 />', '<output value="2&lt;3" />', false);
+            }).to.throw(Error);
+        });
+
+        it("should fail on escape attribute value with />", function () {
+            chai.expect(function () {
+                eq('<output value="../>.." />', '<output value="..&gt;.." />', false);
```

Comment:
This input is really botched by `fixGTBug`:

`<output value="..&gt;&lt;/output&gt;.." />`